src/pytypes/onefuzztypes/models.py

```diff
@@ -568,6 +568,7 @@ class NodeCommandEnvelope(BaseModel):
 
 class Node(BaseModel):
     timestamp: Optional[datetime] = Field(alias="Timestamp")
+    initialized_at: Optional[datetime]
```

Comment:
Is the idea here that when a node enters `NodeState.init`, that implies that it has been updated/reimaged per the latest VMSS model?

Do we expect this to only get set once, and on re-init we get a whole new `Node` (with e.g. a new `machine_id`)? Should we call that out in a field comment?

Reply:
To your first question - yes, Cheick and I were able to confirm that yesterday. 

I'd expect this to only get set once, yes. I can include a field comment.